app/jobs/ingest_job.rb

```diff
@@ -0,0 +1,17 @@
+class IngestJob < Hyrax::ApplicationJob
+  queue_as Hyrax.config.ingest_queue_name
+
+  after_perform do |job|
+    opts = job.arguments.reverse.detect { |x| x.is_a? Hash } || {}
+    wrapper = job.arguments.first
+    ContentNewVersionEventJob.perform_later(wrapper.file_set, wrapper.user) if opts[:notification]
```

Comment:
@mjgiarlo: The job is invoked for both `create_content` and `update_content`.  Only the latter wants notifications (presumably because it is more targeted than the 100 notifications you might get from a single initial directory upload).  I don't know if it overlaps w/ `BaseActor` or not, but the two distinct uses is what prompted the option/param.